### PR TITLE
Initial split into new repo

### PR DESCRIPTION
**Pull request by jamwaffles**

### CODE REVIEW

**Comment by jamwaffles** [comment]
Argh wait, I didn't realise I needed to update this to latest e-g master - new tinybmp changes just went in!

**Comment by jamwaffles** [comment]
Ok, this is ready now.

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*
```
Comment (rfuest) [inline .dockerignore:1]: Do we need an dockerignore in this repo?

File: Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+[package]
+name = "tinybmp"
+description = "No-std, low memory footprint BMP image loader"
+version = "0.2.3"
+authors = ["James Waples <james@wapl.es>"]
+edition = "2018"
+repository = "https://github.com/embedded-graphics/embedded-graphics/tree/master/tinybmp"
+documentation = "https://docs.rs/tinybmp"
+categories = ["embedded", "no-std"]
```
Comment (rfuest) [inline Cargo.toml:9]:
```suggestion
categories = ["embedded", "no-std", "multimedia::images"]
```

File: Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+keywords = ["graphics", "embedded", "bmp", "bitmap", "image"]
```
Comment (rfuest) [inline Cargo.toml:10]: Should we start adding an embedded-graphics keyword to all crates in the e-g ecosystem? This way we could add a link to https://crates.io/keywords/embedded-graphics in the README to list all e-g related crates. e-t and e-l already include this keyword.

File: LICENSE-MIT [hunk 4]
```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2018 James Waples
```
Comment (rfuest) [inline LICENSE-MIT:1]: Should we update the year?

File: README.tpl [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# TinyBMP
+
+[![Build Status](https://circleci.com/gh/jamwaffles/embedded-graphics/tree/master.svg?style=shield)](https://circleci.com/gh/jamwaffles/embedded-graphics/tree/master)
```
Comment (rfuest) [inline README.tpl:3]: This should link to the tinybmp status instead of the e-g one.

File: justfile [hunk 6]
```diff
@@ -0,0 +1,90 @@
+targets := "arm-unknown-linux-gnueabi armv7-unknown-linux-gnueabihf x86_64-unknown-linux-gnu x86_64-unknown-linux-musl thumbv6m-none-eabi thumbv7em-none-eabi thumbv7em-none-eabihf thumbv7m-none-eabi"
```
Comment (rfuest) [inline justfile:1]: The suggestions for `tinytga` also apply to this justfile, except for the comments that contain `tinytga`.

File: release.toml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+no-dev-version = true
+pre-release-replacements = [
+  {file="CHANGELOG.md", search="[Uu]nreleased", replace="{{version}}"},
+  {file="CHANGELOG.md", search="\\.\\.\\.HEAD", replace="...{{tag_name}}"},
+  {file="CHANGELOG.md", search="ReleaseDate", replace="{{date}}"},
+  {file="CHANGELOG.md", search="<!-- next-header -->", replace="<!-- next-header -->\n\n## [Unreleased] - ReleaseDate"},
+  {file="CHANGELOG.md", search="<!-- next-url -->", replace="<!-- next-url -->\n[unreleased]: https://github.com/jamwaffles/{{crate_name}}/compare/{{tag_name}}...HEAD"},
```
Comment (rfuest) [inline release.toml:7]:
```suggestion
  {file="CHANGELOG.md", search="<!-- next-url -->", replace="<!-- next-url -->\n[unreleased]: https://github.com/embedded-graphics/{{crate_name}}/compare/{{tag_name}}...HEAD"},
```

**Review by rfuest — CHANGES_REQUESTED** [review]
The requested changes are pretty much identical with the `tinytga` ones.

File: Cargo.toml (hunk 3 above)
Comment (jamwaffles) [reply Cargo.toml:10]: Sounds good to me. As there's a 5 keyword limit, I'll remove `embedded` and replace it with `embedded-graphics`.

**Review by rfuest — CHANGES_REQUESTED** [review]
The new suggested changes in the `tinytga` repo also apply to this repo.

**Review by rfuest — APPROVED** [review]
LGTM

**Pull request merged**